tool/tsh/db.go

```diff
@@ -135,59 +135,57 @@ func (l databaseListings) Swap(i, j int) {
 }
 
 func listDatabasesAllClusters(cf *CLIConf) error {
```

Comment:
`tsh db ls -R` doesn't work for me with these changes. I tried `tsh db ls -Rv` but it doesn't print anything either.